### PR TITLE
feat: Ability to configure cache folder

### PR DESCRIPTION
**Pull request by srosato**
## Summary/ Motivation (TLDR;)

Now possible to specify cache folder by either 

* using `preview.debug({cacheFolder: 'your-cache-folder'})` 
* automatically with the auto preview feature.

## Related issues

- https://github.com/nvh95/jest-preview/issues/301)
- https://github.com/nvh95/jest-preview/issues/126

## Features

- Specify cache folder for the `debug` function
- Specify cache folder for the `jestPreviewConfigure`

## Fixes

- Running single tests with IDE such as WebStorm/IntelliJ and having the html output being stored at one place instead of relative to the closest jest config it can find.

## Chores

- I ran prettier + eslint and it fixed files outside this PR scope

## Usage

Take a look at `examples/nextjs-nx/apps/app/jest.config.js`, `examples/nextjs-nx/apps/app/jest.setup.ts` and ``examples/nextjs-nx/apps/app/specs/index.spec.tsx`.

Basically:

### For auto-preview

```typescript
jestPreviewConfigure({
  autoPreview: true,
  cacheFolder: 'path-to-custom-cache-folder',
});
```

### Using `debug`

```typescript
// it is best to create a utility file so we do not have to specify it each time we import `debug`

import preview from "jest-preview";

export const debug = () => {
  preview.debug({cacheFolder: 'path-to-custom-cache-folder'});
};
```

### CODE REVIEW

**Comment by srosato** [comment]
Main branch seems to break also on Jest Preview Ecosystem CI / build, not sure how to resolve

**Comment by srosato** [comment]
@nvh95 I wonder what you think about this feature? Is anything more needed? Let me know :)
